fs_brokers/apache_hdfs_broker/src/hadoop-cos-shaded/pom.xml

```diff
@@ -44,6 +44,12 @@ under the License.
             <groupId>com.qcloud.cos</groupId>
             <artifactId>hadoop-cos</artifactId>
             <version>3.3.0-8.3.7</version>
+            <exclusions>
```

Comment:
I think you can bump it to `3.3.3-8.3.10` first, it maybe fixed cve.

Reply:
Let me check that

Reply:
@Smith-Cruise could you please review this again?